homeassistant/components/osramlightify/light.py

```diff
@@ -74,7 +74,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     try:
         bridge = Lightify(host, log_level=logging.NOTSET)
     except OSError as err:
-        msg = "Error connecting to bridge: {} due to: {}".format(host, str(err))
+        msg = f"Error connecting to bridge: {host} due to: {err}"
```

Comment:
Maybe put this directly into the log message? This variable isn't used after it.